src/solver/newton/newton.hpp

```diff
@@ -34,8 +34,22 @@ namespace newton {
  * @{
  */
 
-static constexpr int MAX_ITER = 1e3;
-static constexpr double EPS = 1e-12;
+static constexpr int MAX_ITER = 50;
+static constexpr double EPS = 1e-13;
```

Comment:
Why the change here?

Reply:
The one for MAX_ITER is a compromise between getting closer to NRN and allowing infinite guessing. If Newton converges it converges extremely fast, in every iteration the number of `0` doubles. The problem tends to be that it doesn't converge. However, if it doesn't more iterations are unlikely to help. Either one's stuck just above EPS, or it's paying ping pong, or it's diverging.

The EPS is needed to make the tests pass the check that the residual is under a certain threshold.

Reply:
Was just wondering why you lowered the EPS. Thanks for elaborating on the much lower MAX_ITER.